Manpage fixes applied to X.Org git after libX11-1.1.3 was released.
These should all be included in libX11-1.1.4 so this patch can be
removed when upgrading to that release.

From e3430616f26b68e1439143cbe10732f3fc329d20 Mon Sep 17 00:00:00 2001
From: Eric S. Raymond <esr@thyrsus.com>
Date: Tue, 2 Jan 2007 08:40:00 -0800
Subject: [PATCH] Bug #9516: Markup error in XAllocWMHints.3x

X.Org Bugzilla #9516 <https://bugs.freedesktop.org/show_bug.cgi?id=9516>
---
 man/XAllocWMHints.man |    3 +--
 1 files changed, 1 insertions(+), 2 deletions(-)

diff --git a/man/XAllocWMHints.man b/man/XAllocWMHints.man
index 4c3fe1d..089dc52 100644
--- a/man/XAllocWMHints.man
+++ b/man/XAllocWMHints.man
@@ -46,8 +46,7 @@
 .\"
 .ds xT X Toolkit Intrinsics \- C Language Interface
 .ds xW Athena X Widgets \- C Language X Toolkit Interface
-.ds xL Xlib \- C Language X Interface, \fRO'Reilly and Associates, Sebastopol,
-1991. 
+.ds xL Xlib \- C Language X Interface, \fRO'Reilly and Associates, Sebastopol, 1991. 
 .ds xC Inter-Client Communication Conventions Manual
 .na
 .de Ds
-- 
1.4.1

From ca5d9a625ea0965853fa9e74a448b8c29c78ec95 Mon Sep 17 00:00:00 2001
From: Eric S. Raymond <esr@thyrsus.com>
Date: Tue, 2 Jan 2007 10:18:00 -0800
Subject: [PATCH] Bug 9523: Markup problems in XQueryExtension.3x

X.Org Bugzilla #9523 <https://bugs.freedesktop.org/show_bug.cgi?id=9523>
---
 man/XQueryExtension.man |    3 +--
 1 files changed, 1 insertions(+), 2 deletions(-)

diff --git a/man/XQueryExtension.man b/man/XQueryExtension.man
index a81ec0c..bfc0964 100644
--- a/man/XQueryExtension.man
+++ b/man/XQueryExtension.man
@@ -14,7 +14,7 @@
 .\" of the information in this document.  It is provided \`\`as-is'' without
 .\" express or implied warranty.
 .\"
-.dS xL Programming with Xlib
+.ds xL Programming with Xlib
 .TH XQueryExtension __libmansuffix__ __xorgversion__ "X FUNCTIONS"
 .SH NAME
 XQueryExtension, XListExtensions, XFreeExtensionList \- list available extensions
@@ -25,7 +25,6 @@ Bool XQueryExtension(\^\fIdisplay, \fIna
 char **XListExtensions(\^\fIdisplay\fP, \fInextensions_return\fP\^)
 .HP
 XFreeExtensionList(\^\fIlist\fP\^)
-.FN
 .SH ARGUMENTS
 .IP \fIdisplay\fP 1i
 Specifies the connection to the X server.
-- 
1.4.1

From c316aaf0aab06951db9dc5c9c1148bfea835d885 Mon Sep 17 00:00:00 2001
From: Eric S. Raymond <esr@thyrsus.com>
Date: Sun, 14 Jan 2007 10:38:00 -0800
Subject: [PATCH] Bug #9650: Bad markup in XLoadFont.3x manual page

X.Org Bugzilla #9650 <https://bugs.freedesktop.org/show_bug.cgi?id=9650>
---
 man/XLoadFont.man |   47 +++++++++++++++++++++++------------------------
 1 files changed, 23 insertions(+), 24 deletions(-)

diff --git a/man/XLoadFont.man b/man/XLoadFont.man
index ab7e890..906f20e 100644
--- a/man/XLoadFont.man
+++ b/man/XLoadFont.man
@@ -141,6 +141,16 @@
 .el .sp 10p
 ..
 .ny0
+.de EX
+.sp
+.nf
+.ft CW
+..
+.de EE
+.ft R
+.fi
+.sp
+..
 .TH XLoadFont __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XLoadFont, XQueryFont, XLoadQueryFont, XFreeFont, XGetFontProperty, XUnloadFont, XCharStruct, XFontProp, XChar2b, XFontStruct \- load or unload fonts and font metric structures
@@ -309,9 +319,7 @@ and
 structures contain:
 .LP
 .IN "XCharStruct" "" "@DEF@"
-.Ds 0
-.TA .5i 3i
-.ta .5i 3i
+.EX
 typedef struct {
 	short lbearing;	/\(** origin to left edge of raster */
 	short rbearing;	/\(** origin to right edge of raster */
@@ -320,32 +328,26 @@ typedef struct {
 	short descent;	/\(** baseline to bottom edge of raster */
 	unsigned short attributes;	/\(** per char flags (not predefined) */
 } XCharStruct;
-.De
+.EE
 .LP
 .IN "XFontProp" "" "@DEF@"
-.Ds 0
-.TA .5i 1i 3i
-.ta .5i 1i 3i
+.EX
 typedef struct {
 	Atom	name;
 	unsigned long card32;
 } XFontProp;
-.De
+.EE
 .LP
 .IN "XChar2b" "" "@DEF@"
-.Ds 0
-.TA .5i 3i
-.ta .5i 3i
+.EX
 typedef struct {	/\(** normal 16 bit characters are two bytes */
     unsigned char byte1;
     unsigned char byte2;
 } XChar2b;
-.De
+.EE
 .LP
 .IN "XFontStruct" "" "@DEF@"
-.Ds 0
-.TA .5i 3i
-.ta .5i 3i
+.EX
 typedef struct {
 	XExtData *ext_data;	/\(** hook for extension to hang data */
 	Font fid;	/\(** Font id for this font */
@@ -364,7 +366,7 @@ typedef struct {
 	int ascent;	/\(** logical extent above baseline for spacing */
 	int descent;	/\(** logical decent below baseline for spacing */
 } XFontStruct;
-.De
+.EE
 .LP
 X supports single byte/character, two bytes/character matrix,
 and 16-bit character text operations.
@@ -416,19 +418,16 @@ min_char_or_byte2 and max_char_or_byte2 
 and the 2-byte character index values corresponding to the
 per_char array element N (counting from 0) are:
 .IP
-.nf
+.EX
 	byte1 = N/D + min_byte1
-.br
 	byte2 = N\\D + min_char_or_byte2
-.IP
-.fi
+.EE
 where:
-.IP
-.nf
+.EX
         D = max_char_or_byte2 \- min_char_or_byte2 + 1
         / = integer division
-        \\ = integer modulus
-.fi
+        \e\e = integer modulus
+.EE
 .IP \(bu 5
 If the per_char pointer is NULL, 
 all glyphs between the first and last character indexes
-- 
1.4.1

From 2e7e0748d353d7f53bbd65ec6bf0df8758528ddc Mon Sep 17 00:00:00 2001
From: Eric S. Raymond <esr@thyrsus.com>
Date: Sun, 14 Jan 2007 10:39:00 -0800
Subject: [PATCH] Bug #9651: Bad markup in XcmsColor.3x

X.Org Bugzilla #9651 <https://bugs.freedesktop.org/show_bug.cgi?id=9651>
---
 man/XcmsColor.man |   72 +++++++++++++++++++++++------------------------------
 1 files changed, 31 insertions(+), 41 deletions(-)

diff --git a/man/XcmsColor.man b/man/XcmsColor.man
index 43635ce..8c7d0e9 100644
--- a/man/XcmsColor.man
+++ b/man/XcmsColor.man
@@ -140,6 +140,16 @@
 .el .sp 10p
 ..
 .ny0
+.de EX
+.sp
+.nf
+.ft CW
+..
+.de EE
+.ft R
+.fi
+.sp
+..
 .TH XcmsColor __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XcmsColor, XcmsRGB, XcmsRGBi, XcmsCIEXYZ, XcmsCIEuvY, XcmsCIExyY, XcmsCIELab, XcmsCIELuv, XcmsTekHVC, XcmsPad \- Xcms color struture
@@ -148,9 +158,7 @@ The structure for
 .ZN XcmsColor
 contains:
 .LP
-.Ds 0
-.TA .5i 1i 2.5i
-.ta .5i 1i 2.5i
+.EX
 typedef unsigned long XcmsColorFormat;			/\(** Color Specification Format */
 
 typedef struct {
@@ -168,11 +176,9 @@ typedef struct {
 	unsigned long pixel;
 	XcmsColorFormat format;
 } XcmsColor;			/\(** Xcms Color Structure */
-.De
+.EE
 .LP
-.Ds 0
-.TA .5i 2.5i
-.ta .5i 2.5i
+.EX
 typedef double XcmsFloat;
 
 typedef struct {
@@ -180,88 +186,72 @@ typedef struct {
 	unsigned short green;	/\(** 0x0000 to 0xffff */
 	unsigned short blue;	/\(** 0x0000 to 0xffff */
 } XcmsRGB;		/\(** RGB Device */
-.De
+.EE
 .LP
-.Ds 0
-.TA .5i 2.5i
-.ta .5i 2.5i
+.EX
 typedef struct {
 	XcmsFloat red;	/\(** 0.0 to 1.0 */
 	XcmsFloat green;	/\(** 0.0 to 1.0 */
 	XcmsFloat blue;	/\(** 0.0 to 1.0 */
 } XcmsRGBi;		/\(** RGB Intensity */
-.De
+.EE
 .LP
-.Ds 0
-.TA .5i 2.5i
-.ta .5i 2.5i
+.EX
 typedef struct {
 	XcmsFloat X;
 	XcmsFloat Y;	/\(** 0.0 to 1.0 */
 	XcmsFloat Z;
 } XcmsCIEXYZ;		/\(** CIE XYZ */
-.De
+.EE
 .LP
-.Ds 0
-.TA .5i 2.5i
-.ta .5i 2.5i
+.EX
 typedef struct {
 	XcmsFloat u_prime;	/\(** 0.0 to ~0.6 */
 	XcmsFloat v_prime;	/\(** 0.0 to ~0.6 */
 	XcmsFloat Y; 	/\(** 0.0 to 1.0 */
 } XcmsCIEuvY;		/\(** CIE u'v'Y */
-.De
+.EE
 .LP
-.Ds 0
-.TA .5i 2.5i
-.ta .5i 2.5i
+.EX
 typedef struct {
 	XcmsFloat x; 	/\(** 0.0 to ~.75 */
 	XcmsFloat y; 	/\(** 0.0 to ~.85 */
 	XcmsFloat Y; 	/\(** 0.0 to 1.0 */
 } XcmsCIExyY;		/\(** CIE xyY */
-.De
+.EE
 .LP
-.Ds 0
-.TA .5i 2.5i
-.ta .5i 2.5i
+.EX
 typedef struct {
 	XcmsFloat L_star; 	/\(** 0.0 to 100.0 */
 	XcmsFloat a_star;
 	XcmsFloat b_star;
 } XcmsCIELab;		/\(** CIE L*a*b* */
-.De
+.EE
 .LP
-.Ds 0
-.TA .5i 2.5i
-.ta .5i 2.5i
+.EX
 typedef struct {
 	XcmsFloat L_star; 	/\(** 0.0 to 100.0 */
 	XcmsFloat u_star;
 	XcmsFloat v_star;
 } XcmsCIELuv;		/\(** CIE L*u*v* */
-.De
+.EE
 .LP
-.Ds 0
-.TA .5i 2.5i
-.ta .5i 2.5i
+.EX
 typedef struct {
 	XcmsFloat H; 	/\(** 0.0 to 360.0 */
 	XcmsFloat V; 	/\(** 0.0 to 100.0 */
 	XcmsFloat C; 	/\(** 0.0 to 100.0 */
 } XcmsTekHVC;		/\(** TekHVC */
-.De
+.EE
 .LP
-.Ds 0
-.TA .5i 2.5i
-.ta .5i 2.5i
+.EX
 typedef struct {
 	XcmsFloat pad0;
 	XcmsFloat pad1;
 	XcmsFloat pad2;
 	XcmsFloat pad3;
 } XcmsPad;		/\(** four doubles */
-.De
+.EE
 .SH DESCRIPTION
 The
 .ZN XcmsColor
@@ -271,5 +261,5 @@ each supporting color specification enco
 XcmsAllocColor(3X11),
 XcmsStoreColor(3X11),
 XcmsConvertColors(3X11),
-.br
+.LP
 \fI\*(xL\fP
-- 
1.4.1

From 9d3ceea4b902e0471824c1e07ad64342b9a1114e Mon Sep 17 00:00:00 2001
From: Eric S. Raymond <esr@thyrsus.com>
Date: Sun, 14 Jan 2007 10:40:00 -0800
Subject: [PATCH] Bug #9652: Bad markup in XDrawArc.3x

X.Org Bugzilla #9652 <https://bugs.freedesktop.org/show_bug.cgi?id=9652>
---
 man/XDrawArc.man |   23 +++++++++++++++--------
 1 files changed, 15 insertions(+), 8 deletions(-)

diff --git a/man/XDrawArc.man b/man/XDrawArc.man
index c878c45..00a79cf 100644
--- a/man/XDrawArc.man
+++ b/man/XDrawArc.man
@@ -142,6 +142,16 @@
 .el .sp 10p
 ..
 .ny0
+.de EX
+.sp
+.nf
+.ft CW
+..
+.de EE
+.ft R
+.fi
+.sp
+..
 '\" e
 .TH XDrawArc __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
@@ -240,13 +250,12 @@ than in degrees scaled by 64) in the ran
 returns a value in the range % [ - pi over 2 , ~pi over 2  ] %
 and adjust is:
 .LP
-.Ds
-.TA 1i 2i
-.ta 1i 2i
+.TS
+l l.
 %0%	for normal-angle in the range % [ 0 , ~pi over 2  ]%
 %pi%	for normal-angle in the range % [ pi over 2 , ~{3 pi} over 2  ]%
 %2 pi%	for normal-angle in the range % [ {3 pi} over 2 , ~2 pi  ]%
-.De
+.TE
 .LP
 For any given arc, 
 .ZN XDrawArc
@@ -295,15 +304,13 @@ The
 .ZN XArc
 structure contains:
 .LP
-.Ds 0
-.TA .5i 3i
-.ta .5i 3i
+.EX
 typedef struct {
 	short x, y;
 	unsigned short width, height;
 	short angle1, angle2;             /\(** Degrees * 64 */
 } XArc;
-.De
+.EE
 .LP
 All x and y members are signed integers.
 The width and height members are 16-bit unsigned integers.
-- 
1.4.1

From 2db713252090cae08f0200fecad4fc25fb64c8b1 Mon Sep 17 00:00:00 2001
From: Eric S. Raymond <esr@thyrsus.com>
Date: Sun, 14 Jan 2007 10:40:00 -0800
Subject: [PATCH] Bug #9653: Bad markup in XQueryColor.3x

X.Org Bugzilla #9653 <https://bugs.freedesktop.org/show_bug.cgi?id=9653>
---
 man/XQueryColor.man |   10 ++++------
 1 files changed, 4 insertions(+), 6 deletions(-)

diff --git a/man/XQueryColor.man b/man/XQueryColor.man
index eff8a6c..e2e884b 100644
--- a/man/XQueryColor.man
+++ b/man/XQueryColor.man
@@ -1,3 +1,4 @@
+'\" t
 .\" Copyright \(co 1985, 1986, 1987, 1988, 1989, 1990, 1991, 1994, 1996 X Consortium
 .\"
 .\" Permission is hereby granted, free of charge, to any person obtaining
@@ -276,16 +277,13 @@ supported, but its continued use is not 
 The syntax is an initial sharp sign character followed by
 a numeric specification, in one of the following formats:
 .LP
-.\" Start marker code here
-.Ds 0
-.TA 2i
-.ta 2i
+.TS
+l l.
 \&#RGB	(4 bits each)
 \&#RRGGBB	(8 bits each)
 \&#RRRGGGBBB	(12 bits each)
 \&#RRRRGGGGBBBB	(16 bits each)
-.De
-.\" End marker code here
+.TE
 .LP
 The R, G, and B represent single hexadecimal digits.
 When fewer than 16 bits each are specified, 
-- 
1.4.1

From 4341d1a34b2a2e460b58131b6fd81935f3355bbc Mon Sep 17 00:00:00 2001
From: Eric S. Raymond <esr@thyrsus.com>
Date: Sun, 14 Jan 2007 10:43:00 -0800
Subject: [PATCH] Bug #9654: Bad markup in XrmGetFileDatabase.3x

X.Org Bugzilla #9654 <https://bugs.freedesktop.org/show_bug.cgi?id=9654>
---
 man/XrmGetFileDatabase.man |   17 ++++++++++++-----
 1 files changed, 12 insertions(+), 5 deletions(-)

diff --git a/man/XrmGetFileDatabase.man b/man/XrmGetFileDatabase.man
index 959603a..c46994a 100644
--- a/man/XrmGetFileDatabase.man
+++ b/man/XrmGetFileDatabase.man
@@ -140,6 +140,16 @@
 .el .sp 10p
 ..
 .ny0
+.de EX
+.sp
+.nf
+.ft CW
+..
+.de EE
+.ft R
+.fi
+.sp
+..
 .TH XrmGetFileDatabase __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XrmGetFileDatabase, XrmPutFileDatabase, XrmGetStringDatabase, XrmLocaleOfDatabase, XrmGetDatabase, XrmSetDatabase, XrmDestroyDatabase \- retrieve and store resource databases
@@ -247,9 +257,7 @@ terminated by newline characters or the 
 The syntax of an individual resource line is:
 .LP
 .\" Start marker code here
-.Ds 0
-.TA 1.5i 1.75i
-.ta 1.5i 1.75i
+.EX
 ResourceLine	=	Comment | IncludeFile | ResourceSpec | <empty line>
 Comment	=	"!" {<any character except null or newline>}
 IncludeFile	=	"#" WhiteSpace "include" WhiteSpace FileName WhiteSpace
@@ -262,8 +270,7 @@ Component	=	"?" | ComponentName
 ComponentName	=	NameChar {NameChar}
 NameChar	=	"a"\-"z" | "A"\-"Z" | "0"\-"9" | "_" | "-"
 Value	=	{<any character except null or unescaped newline>}
-.De
-.\" End marker code here
+.EE
 .LP
 Elements separated by vertical bar (|) are alternatives.
 Curly braces ({\&.\&.\&.}) indicate zero or more repetitions
-- 
1.4.1

From 74cba78daa738ef4d92096107d95c1a585933666 Mon Sep 17 00:00:00 2001
From: Eric S. Raymond <esr@thyrsus.com>
Date: Sun, 14 Jan 2007 10:43:00 -0800
Subject: [PATCH] Bug #9655: Bad markup in XrmUniqueQuark.3x

X.Org Bugzilla #9655 <https://bugs.freedesktop.org/show_bug.cgi?id=9655>
---
 man/XrmUniqueQuark.man |   16 +++++++---------
 1 files changed, 7 insertions(+), 9 deletions(-)

diff --git a/man/XrmUniqueQuark.man b/man/XrmUniqueQuark.man
index 26332be..baaeac9 100644
--- a/man/XrmUniqueQuark.man
+++ b/man/XrmUniqueQuark.man
@@ -143,7 +143,7 @@
 .TH XrmUniqueQuark __libmansuffix__ __xorgversion__ "XLIB FUNCTIONS"
 .SH NAME
 XrmUniqueQuark, XrmStringToQuark, XrmPermStringToQuark, XrmQuarkToString, XrmStringToQuarkList, XrmStringToBindingQuarkList \- manipulate resource quarks
-.SH SYNTAX
+.SH SYNOPSIS
 .HP
 XrmQuark XrmUniqueQuark\^(void); 
 .LP
@@ -267,9 +267,9 @@ A binding list is a list of type
 and indicates if components of name or class lists are bound tightly or loosely
 (that is, if wildcarding of intermediate components is specified).
 .LP
-.Ds 0
+.EX
 typedef enum {XrmBindTightly, XrmBindLoosely} XrmBinding, *XrmBindingList;
-.De
+.EE
 .LP
 .ZN XrmBindTightly
 indicates that a period separates the components, and
@@ -290,17 +290,15 @@ If the string does not start with period
 a period is assumed.
 For example, ``*a.b*c'' becomes:
 .LP
-.Ds 0
-.TA .75i 1.5i 2.25i
-.ta .75i 1.5i 2.25i
+.TS
+l l l.
 quarks	a	b	c
-.br
 bindings	loose	tight	loose
-.De
+.TE
 .SH "SEE ALSO"
 XrmGetResource(3X11),
 XrmInitialize(3X11),
 XrmMergeDatabases(3X11),
 XrmPutResource(3X11)
-.br
+.LP
 \fI\*(xL\fP
-- 
1.4.1

From fa4effe82759f864a22a2dc6c920fa72ddb175a8 Mon Sep 17 00:00:00 2001
From: Eric S. Raymond <esr@thyrsus.com>
Date: Sun, 14 Jan 2007 10:48:00 -0800
Subject: [PATCH] Bug #9658: Bad markup on XIfEvent.3x

X.Org Bugzilla #9658 <https://bugs.freedesktop.org/show_bug.cgi?id=9658>
---
 man/XIfEvent.man |    2 --
 1 files changed, 0 insertions(+), 2 deletions(-)

diff --git a/man/XIfEvent.man b/man/XIfEvent.man
index 61d28e5..17c483b 100644
--- a/man/XIfEvent.man
+++ b/man/XIfEvent.man
@@ -221,7 +221,6 @@ has first called
 .ZN XLockDisplay .
 .LP
 The predicate procedure and its associated arguments are:
-.sM
 .HP
 Bool (\^*\fIpredicate\fP\^)\^(\^Display *\fIdisplay\fP, XEvent *\fIevent\fP, XPointer \fIarg\fP\^)
 .IP \fIdisplay\fP 1i
@@ -238,7 +237,6 @@ or
 .ZN XPeekIfEvent 
 function.
 .LP
-.eM
 The predicate procedure is called once for each
 event in the queue until it finds a match. 
 After finding a match, the predicate procedure must return 
-- 
1.4.1

From 11ea09745efa8de7dc82fe30ebd2393f08390957 Mon Sep 17 00:00:00 2001
From: Eric Anholt <eric@anholt.net>
Date: Fri, 31 Aug 2007 17:30:33 -0700
Subject: [PATCH] Bug #2081: Note the range limitation of XSetScreenSaver arguments.
---
 man/XSetScreenSaver.man |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/man/XSetScreenSaver.man b/man/XSetScreenSaver.man
index c0c1e36..4a42bcf 100644
--- a/man/XSetScreenSaver.man
+++ b/man/XSetScreenSaver.man
@@ -213,6 +213,8 @@ If the timeout value is nonzero, 
 .ZN XSetScreenSaver
 enables the screen saver.
 An interval of 0 disables the random-pattern motion.
+Both values are limited to a 16-bit signed integer range by the wire protocol,
+despite the C prototype.
 If no input from devices (keyboard, mouse, and so on) is generated 
 for the specified number of timeout seconds once the screen saver is enabled,
 the screen saver is activated.
-- 
1.4.1

From 6e5485e0a5e3ab738becad12193e760c5fee83a4 Mon Sep 17 00:00:00 2001
From: Samuel Thibault <samuel.thibault@ens-lyon.org>
Date: Sun, 28 Oct 2007 04:44:00 -0800
Subject: [PATCH] X.Org Bug #12983: Typos in ./man/XChangeKeyboardControl.man

<http://bugs.freedesktop.org/show_bug.cgi?id=12983>
---
 man/XChangeKeyboardControl.man |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/man/XChangeKeyboardControl.man b/man/XChangeKeyboardControl.man
index 58a5c9b..6b05848 100644
--- a/man/XChangeKeyboardControl.man
+++ b/man/XChangeKeyboardControl.man
@@ -149,9 +149,9 @@ XChangeKeyboardControl, XGetKeyboardCont
 .SH SYNTAX
 .HP
 int XChangeKeyboardControl\^(\^Display *\fIdisplay\fP, unsigned long
-\fIvalue_mask\fP\^, XKeyBoardControl *\fIvalues\fP\^); 
+\fIvalue_mask\fP\^, XKeyboardControl *\fIvalues\fP\^); 
 .HP
-int XGetKeyboardControl\^(\^Display *\fIdisplay\fP, XKeyBoardState
+int XGetKeyboardControl\^(\^Display *\fIdisplay\fP, XKeyboardState
 *\fIvalues_return\fP\^); 
 .HP
 int XAutoRepeatOn\^(\^Display *\fIdisplay\fP\^); 
-- 
1.4.1

From 4b91ed099554626f1ec17d5bdf7bd77ce1a70037 Mon Sep 17 00:00:00 2001
From: Alan Coopersmith <alan.coopersmith@sun.com>
Date: Wed, 21 Nov 2007 16:50:04 -0800
Subject: [PATCH] X.Org Bug #4312: incorrect comment asterisk in XAnyEvent.3x man page

X.Org Bugzilla #4312 <http://bugs.freedesktop.org/show_bug.cgi?id=4312>

Protect /* sequences from cpp pre-processor removal without transforming
to Unicode mathematical asterisk character
---
 man/XAddHost.man                  |   14 +++----
 man/XAllocSizeHints.man           |   32 ++++++++-------
 man/XAllocStandardColormap.man    |    4 +-
 man/XAllocWMHints.man             |   26 ++++++------
 man/XAnyEvent.man                 |    8 ++--
 man/XButtonEvent.man              |   78 +++++++++++++++++++------------------
 man/XChangeKeyboardControl.man    |    8 ++--
 man/XChangeKeyboardMapping.man    |    4 +-
 man/XCirculateEvent.man           |   10 ++---
 man/XCirculateRequestEvent.man    |   10 ++---
 man/XClientMessageEvent.man       |    8 ++--
 man/XColormapEvent.man            |   12 +++---
 man/XConfigureEvent.man           |    8 ++--
 man/XConfigureRequestEvent.man    |   10 ++---
 man/XConfigureWindow.man          |    4 +-
 man/XCreateColormap.man           |    6 +--
 man/XCreateGC.man                 |   44 ++++++++++-----------
 man/XCreateWindow.man             |   34 ++++++++--------
 man/XCreateWindowEvent.man        |   20 +++++----
 man/XCrossingEvent.man            |   30 +++++++-------
 man/XDestroyWindowEvent.man       |    8 ++--
 man/XDrawArc.man                  |    2 -
 man/XDrawText.man                 |   16 ++++----
 man/XErrorEvent.man               |   12 +++---
 man/XExposeEvent.man              |   10 ++---
 man/XFocusChangeEvent.man         |   14 +++----
 man/XFontSetExtents.man           |    4 +-
 man/XGetVisualInfo.man            |    4 +-
 man/XGetWindowAttributes.man      |   42 ++++++++++----------
 man/XGraphicsExposeEvent.man      |   26 ++++++------
 man/XGravityEvent.man             |    8 ++--
 man/XKeymapEvent.man              |   10 ++---
 man/XLoadFont.man                 |   46 +++++++++++-----------
 man/XMapEvent.man                 |   26 ++++++------
 man/XMapRequestEvent.man          |    8 ++--
 man/XPropertyEvent.man            |   10 ++---
 man/XReparentEvent.man            |    8 ++--
 man/XResizeRequestEvent.man       |    8 ++--
 man/XSelectionClearEvent.man      |    8 ++--
 man/XSelectionEvent.man           |   10 ++---
 man/XSelectionRequestEvent.man    |    8 ++--
 man/XStringListToTextProperty.man |    8 ++--
 man/XUnmapEvent.man               |    8 ++--
 man/XVisibilityEvent.man          |    8 ++--
 man/XcmsColor.man                 |   58 ++++++++++++++--------------
 man/XmbDrawText.man               |   16 ++++----
 man/XmbTextListToTextProperty.man |   18 ++++-----
 man/XrmInitialize.man             |   24 ++++++-----
 48 files changed, 399 insertions(+), 399 deletions(-)

diff --git a/man/XAddHost.man b/man/XAddHost.man
index b9f0aa7..fe95c7d 100644
--- a/man/XAddHost.man
+++ b/man/XAddHost.man
@@ -305,9 +305,9 @@ structure contains:
 .TA .5i 3i
 .ta .5i 3i
 typedef struct {
-	int family;	/\(** for example FamilyInternet */
-	int length;	/\(** length of address, in bytes */
-	char *address;	/\(** pointer to where to find the address */
+	int family;	/\&* for example FamilyInternet */
+	int length;	/\&* length of address, in bytes */
+	char *address;	/\&* pointer to where to find the address */
 } XHostAddress;
 .De
 .LP
@@ -331,10 +331,10 @@ structure which contains:
 .TA .5i 3i
 .ta .5i 3i
 typedef struct {
-	int typelength;		/\(** length of type string, in bytes */
-	int valuelength;	/\(** length of value string, in bytes */
-	char *type;		/\(** pointer to where to find the type string */
-	char *value;		/\(** pointer to where to find the address */
+	int typelength;		/\&* length of type string, in bytes */
+	int valuelength;	/\&* length of value string, in bytes */
+	char *type;		/\&* pointer to where to find the type string */
+	char *value;		/\&* pointer to where to find the address */
 } XServerInterpretedAddress;
 .De
 .LP
diff --git a/man/XAllocSizeHints.man b/man/XAllocSizeHints.man
index e957214..226b9d5 100644
--- a/man/XAllocSizeHints.man
+++ b/man/XAllocSizeHints.man
@@ -324,7 +324,7 @@ The
 .ZN XSizeHints
 structure contains:
 .LP
-/\(** Size hints mask bits */
+/\&* Size hints mask bits */
 .TS
 lw(.5i) lw(1.1i) lw(1.5i) lw(3.1i).
 T{
@@ -334,7 +334,7 @@ T}	T{
 T}	T{
 (1L << 0)
 T}	T{
-/\(** user specified x, y */
+/\&* user specified x, y */
 T}
 T{
 \&#define
@@ -343,7 +343,7 @@ T}	T{
 T}	T{
 (1L << 1)
 T}	T{
-/\(** user specified width, height */
+/\&* user specified width, height */
 T}
 T{
 \&#define
@@ -352,7 +352,7 @@ T}	T{
 T}	T{
 (1L << 2)
 T}	T{
-/\(** program specified position */
+/\&* program specified position */
 T}
 T{
 \&#define
@@ -361,7 +361,7 @@ T}	T{
 T}	T{
 (1L << 3)
 T}	T{
-/\(** program specified size */
+/\&* program specified size */
 T}
 T{
 \&#define
@@ -370,7 +370,7 @@ T}	T{
 T}	T{
 (1L << 4)
 T}	T{
-/\(** program specified minimum size */
+/\&* program specified minimum size */
 T}
 T{
 \&#define
@@ -379,7 +379,7 @@ T}	T{
 T}	T{
 (1L << 5)
 T}	T{
-/\(** program specified maximum size */
+/\&* program specified maximum size */
 T}
 T{
 \&#define
@@ -388,7 +388,7 @@ T}	T{
 T}	T{
 (1L << 6)
 T}	T{
-/\(** program specified resize increments */
+/\&* program specified resize increments */
 T}
 T{
 \&#define
@@ -397,7 +397,7 @@ T}	T{
 T}	T{
 (1L << 7)
 T}	T{
-/\(** program specified min and max aspect ratios */
+/\&* program specified min and max aspect ratios */
 T}
 T{
 \&#define
@@ -430,22 +430,22 @@ T}
 .Ds 0
 .TA .5i 2.5i
 .ta .5i 2.5i
-/\(** Values */
+/\&* Values */
 
 typedef struct {
-	long flags;	/\(** marks which fields in this structure are defined */
-	int x, y;	/\(** Obsolete */
-	int width, height;	/\(** Obsolete */
+	long flags;	/\&* marks which fields in this structure are defined */
+	int x, y;	/\&* Obsolete */
+	int width, height;	/\&* Obsolete */
 	int min_width, min_height;
 	int max_width, max_height;
 	int width_inc, height_inc;
 	struct {
-	       int x;	/\(** numerator */
-	       int y;	/\(** denominator */
+	       int x;	/\&* numerator */
+	       int y;	/\&* denominator */
 	} min_aspect, max_aspect;
 	int base_width, base_height;
 	int win_gravity;
-	/\(** this structure may be extended in the future */
+	/\&* this structure may be extended in the future */
 } XSizeHints;
 .De
 .LP
diff --git a/man/XAllocStandardColormap.man b/man/XAllocStandardColormap.man
index 29039e5..0baf05c 100644
--- a/man/XAllocStandardColormap.man
+++ b/man/XAllocStandardColormap.man
@@ -284,7 +284,7 @@ The
 .ZN XStandardColormap
 structure contains:
 .LP
-/\(** Hints */
+/\&* Hints */
 .TS
 lw(.5i) lw(2i) lw(1i).
 T{
@@ -295,7 +295,7 @@ T}	T{
 ( (XID) 1L)
 T}
 .TE
-/\(** Values */
+/\&* Values */
 .IN "XStandardColormap" "" "@DEF@"
 .Ds 0
 .TA .5i 2.5i
diff --git a/man/XAllocWMHints.man b/man/XAllocWMHints.man
index 089dc52..9afaa7d 100644
--- a/man/XAllocWMHints.man
+++ b/man/XAllocWMHints.man
@@ -217,7 +217,7 @@ The
 .ZN XWMHints
 structure contains:
 .LP
-/\(** Window manager hints mask bits */
+/\&* Window manager hints mask bits */
 .TS
 lw(.5i) lw(2.5i) lw(2.5i).
 T{
@@ -292,19 +292,19 @@ T}
 .Ds 0
 .TA .5i 2.5i
 .ta .5i 2.5i
-/\(** Values */
+/\&* Values */
 
 typedef struct {
-	long flags;	/\(** marks which fields in this structure are defined */
-	Bool input;	/\(** does this application rely on the window manager to
+	long flags;	/\&* marks which fields in this structure are defined */
+	Bool input;	/\&* does this application rely on the window manager to
 			get keyboard input? */
-	int initial_state;	/\(** see below */
-	Pixmap icon_pixmap;	/\(** pixmap to be used as icon */
-	Window icon_window;	/\(** window to be used as icon */
-	int icon_x, icon_y;	/\(** initial position of icon */
-	Pixmap icon_mask;	/\(** pixmap to be used as mask for icon_pixmap */
-	XID window_group;	/\(** id of related window group */
-	/\(** this structure may be extended in the future */
+	int initial_state;	/\&* see below */
+	Pixmap icon_pixmap;	/\&* pixmap to be used as icon */
+	Window icon_window;	/\&* window to be used as icon */
+	int icon_x, icon_y;	/\&* initial position of icon */
+	Pixmap icon_mask;	/\&* pixmap to be used as mask for icon_pixmap */
+	XID window_group;	/\&* id of related window group */
+	/\&* this structure may be extended in the future */
 } XWMHints;
 .De
 .LP
@@ -358,7 +358,7 @@ T}	T{
 T}	T{
 1
 T}	T{
-/\(** most applications start this way */
+/\&* most applications start this way */
 T}
 T{
 \&#define
@@ -367,7 +367,7 @@ T}	T{
 T}	T{
 3
 T}	T{
-/\(** application wants to start as an icon */
+/\&* application wants to start as an icon */
 T}
 .TE
 The icon_mask specifies which pixels of the icon_pixmap should be used as the
diff --git a/man/XAnyEvent.man b/man/XAnyEvent.man
index f095958..40be01e 100644
--- a/man/XAnyEvent.man
+++ b/man/XAnyEvent.man
@@ -153,9 +153,9 @@ have the following common members:
 .ta .5i 3i
 typedef struct {
 	int type;
-	unsigned long serial;	/\(** # of last request processed by server */
-	Bool send_event;	/\(** true if this came from a SendEvent request */
-	Display *display;	/\(** Display the event was read from */
+	unsigned long serial;	/\&* # of last request processed by server */
+	Bool send_event;	/\&* true if this came from a SendEvent request */
+	Display *display;	/\&* Display the event was read from */
 	Window window;
 } XAnyEvent;
 .De
@@ -186,7 +186,7 @@ structure is a union of the individual s
 .TA .5i 3i
 .ta .5i 3i
 typedef union _XEvent {
-	int type;	/\(** must not be changed */
+	int type;	/\&* must not be changed */
 	XAnyEvent xany;
 	XKeyEvent xkey;
 	XButtonEvent xbutton;
diff --git a/man/XButtonEvent.man b/man/XButtonEvent.man
index 4a28ea8..7cc3e54 100644
--- a/man/XButtonEvent.man
+++ b/man/XButtonEvent.man
@@ -158,19 +158,19 @@ events contain:
 .TA .5i 3i
 .ta .5i 3i
 typedef struct {
-	int type;	/\(** ButtonPress or ButtonRelease */
-	unsigned long serial;	/\(** # of last request processed by server */
-	Bool send_event;	/\(** true if this came from a SendEvent request */
-	Display *display;	/\(** Display the event was read from */
-	Window window;	/\(** ``event'' window it is reported relative to */
-	Window root;	/\(** root window that the event occurred on */
-	Window subwindow;	/\(** child window */
-	Time time;	/\(** milliseconds */
-	int x, y;	/\(** pointer x, y coordinates in event window */
-	int x_root, y_root;	/\(** coordinates relative to root */
-	unsigned int state;	/\(** key or button mask */
-	unsigned int button;	/\(** detail */
-	Bool same_screen;	/\(** same screen flag */
+	int type;	/\&* ButtonPress or ButtonRelease */
+	unsigned long serial;	/\&* # of last request processed by server */
+	Bool send_event;	/\&* true if this came from a SendEvent request */
+	Display *display;	/\&* Display the event was read from */
+	Window window;	/\&* ``event'' window it is reported relative to */
+	Window root;	/\&* root window that the event occurred on */
+	Window subwindow;	/\&* child window */
+	Time time;	/\&* milliseconds */
+	int x, y;	/\&* pointer x, y coordinates in event window */
+	int x_root, y_root;	/\&* coordinates relative to root */
+	unsigned int state;	/\&* key or button mask */
+	unsigned int button;	/\&* detail */
+	Bool same_screen;	/\&* same screen flag */
 } XButtonEvent;
 typedef XButtonEvent XButtonPressedEvent;
 typedef XButtonEvent XButtonReleasedEvent;
@@ -180,19 +180,19 @@ typedef XButtonEvent XButtonReleasedEven
 .TA .5i 3i
 .ta .5i 3i
 typedef struct {
-	int type;	/\(** KeyPress or KeyRelease */
-	unsigned long serial;	/\(** # of last request processed by server */
-	Bool send_event;	/\(** true if this came from a SendEvent request */
-	Display *display;	/\(** Display the event was read from */
-	Window window;	/\(** ``event'' window it is reported relative to */
-	Window root;	/\(** root window that the event occurred on */
-	Window subwindow;	/\(** child window */
-	Time time;	/\(** milliseconds */
-	int x, y;	/\(** pointer x, y coordinates in event window */
-	int x_root, y_root;	/\(** coordinates relative to root */
-	unsigned int state;	/\(** key or button mask */
-	unsigned int keycode;	/\(** detail */
-	Bool same_screen;	/\(** same screen flag */
+	int type;	/\&* KeyPress or KeyRelease */
+	unsigned long serial;	/\&* # of last request processed by server */
+	Bool send_event;	/\&* true if this came from a SendEvent request */
+	Display *display;	/\&* Display the event was read from */
+	Window window;	/\&* ``event'' window it is reported relative to */
+	Window root;	/\&* root window that the event occurred on */
+	Window subwindow;	/\&* child window */
+	Time time;	/\&* milliseconds */
+	int x, y;	/\&* pointer x, y coordinates in event window */
+	int x_root, y_root;	/\&* coordinates relative to root */
+	unsigned int state;	/\&* key or button mask */
+	unsigned int keycode;	/\&* detail */
+	Bool same_screen;	/\&* same screen flag */
 } XKeyEvent;
 typedef XKeyEvent XKeyPressedEvent;
 typedef XKeyEvent XKeyReleasedEvent;
@@ -202,19 +202,19 @@ typedef XKeyEvent XKeyReleasedEvent;
 .TA .5i 3i
 .ta .5i 3i
 typedef struct {
-	int type;	/\(** MotionNotify */
-	unsigned long serial;	/\(** # of last request processed by server */
-	Bool send_event;	/\(** true if this came from a SendEvent request */
-	Display *display;	/\(** Display the event was read from */
-	Window window;	/\(** ``event'' window reported relative to */
-	Window root;	/\(** root window that the event occurred on */
-	Window subwindow;	/\(** child window */
-	Time time;	/\(** milliseconds */
-	int x, y;	/\(** pointer x, y coordinates in event window */
-	int x_root, y_root;	/\(** coordinates relative to root */
-	unsigned int state;	/\(** key or button mask */
-	char is_hint;	/\(** detail */
-	Bool same_screen;	/\(** same screen flag */
+	int type;	/\&* MotionNotify */
+	unsigned long serial;	/\&* # of last request processed by server */
+	Bool send_event;	/\&* true if this came from a SendEvent request */
+	Display *display;	/\&* Display the event was read from */
+	Window window;	/\&* ``event'' window reported relative to */
+	Window root;	/\&* root window that the event occurred on */
+	Window subwindow;	/\&* child window */
+	Time time;	/\&* milliseconds */
+	int x, y;	/\&* pointer x, y coordinates in event window */
+	int x_root, y_root;	/\&* coordinates relative to root */
+	unsigned int state;	/\&* key or button mask */
+	char is_hint;	/\&* detail */
+	Bool same_screen;	/\&* same screen flag */
 } XMotionEvent;
 typedef XMotionEvent XPointerMovedEvent;
 .De
diff --git a/man/XChangeKeyboardControl.man b/man/XChangeKeyboardControl.man
index 6b05848..dd3225d 100644
--- a/man/XChangeKeyboardControl.man
+++ b/man/XChangeKeyboardControl.man
@@ -251,7 +251,7 @@ The
 structure contains:
 .LP
 .LP
-/\(** Mask bits for ChangeKeyboardControl */
+/\&* Mask bits for ChangeKeyboardControl */
 .TS
 lw(.5i) lw(2.5i) lw(.8i).
 T{
@@ -315,7 +315,7 @@ T}
 .Ds 0
 .TA .5i 2.5i
 .ta .5i 2.5i
-/\(** Values */
+/\&* Values */
 
 typedef struct {
 	int key_click_percent;
@@ -323,9 +323,9 @@ typedef struct {
 	int bell_pitch;
 	int bell_duration;
 	int led;
-	int led_mode;	/\(** LedModeOn, LedModeOff */
+	int led_mode;	/\&* LedModeOn, LedModeOff */
 	int key;
-	int auto_repeat_mode;	/\(** AutoRepeatModeOff, AutoRepeatModeOn, 
+	int auto_repeat_mode;	/\&* AutoRepeatModeOff, AutoRepeatModeOn, 
                             	AutoRepeatModeDefault */
 } XKeyboardControl;
 .De
diff --git a/man/XChangeKeyboardMapping.man b/man/XChangeKeyboardMapping.man
index 67b030a..e157ec9 100644
--- a/man/XChangeKeyboardMapping.man
+++ b/man/XChangeKeyboardMapping.man
@@ -428,8 +428,8 @@ structure contains:
 .TA .5i 2.5i
 .ta .5i 2.5i
 typedef struct {
-	int max_keypermod;	/\(** This server's max number of keys per modifier */
-	KeyCode *modifiermap;	/\(** An 8 by max_keypermod array of the modifiers */
+	int max_keypermod;	/\&* This server's max number of keys per modifier */
+	KeyCode *modifiermap;	/\&* An 8 by max_keypermod array of the modifiers */
 } XModifierKeymap;
 .De
 .SH DIAGNOSTICS
diff --git a/man/XCirculateEvent.man b/man/XCirculateEvent.man
index b3e4d8d..1ac71f8 100644
--- a/man/XCirculateEvent.man
+++ b/man/XCirculateEvent.man
@@ -153,13 +153,13 @@ events contains:
 .TA .5i 3i
 .ta .5i 3i
 typedef struct {
-	int type;	/\(** CirculateNotify */
-	unsigned long serial;	/\(** # of last request processed by server */
-	Bool send_event;	/\(** true if this came from a SendEvent request */
-	Display *display;	/\(** Display the event was read from */
+	int type;	/\&* CirculateNotify */
+	unsigned long serial;	/\&* # of last request processed by server */
+	Bool send_event;	/\&* true if this came from a SendEvent request */
+	Display *display;	/\&* Display the event was read from */
 	Window event;
 	Window window;
-	int place;	/\(** PlaceOnTop, PlaceOnBottom */
+	int place;	/\&* PlaceOnTop, PlaceOnBottom */
 } XCirculateEvent;
 .De
 .LP
diff --git a/man/XCirculateRequestEvent.man b/man/XCirculateRequestEvent.man
index ce97c26..6c9ac80 100644
--- a/man/XCirculateRequestEvent.man
+++ b/man/XCirculateRequestEvent.man
@@ -153,13 +153,13 @@ events contains:
 .TA .5i 3i
 .ta .5i 3i
 typedef struct {
-	int type;	/\(** CirculateRequest */
-	unsigned long serial;	/\(** # of last request processed by server */
-	Bool send_event;	/\(** true if this came from a SendEvent request */
-	Display *display;	/\(** Display the event was read from */
+	int type;	/\&* CirculateRequest */
+	unsigned long serial;	/\&* # of last request processed by server */
+	Bool send_event;	/\&* true if this came from a SendEvent request */
+	Display *display;	/\&* Display the event was read from */
 	Window parent;
 	Window window;
-	int place;	/\(** PlaceOnTop, PlaceOnBottom */
+	int place;	/\&* PlaceOnTop, PlaceOnBottom */
 } XCirculateRequestEvent;
 .De
 .LP
diff --git a/man/XClientMessageEvent.man b/man/XClientMessageEvent.man
index 915cc55..da786af 100644
--- a/man/XClientMessageEvent.man
+++ b/man/XClientMessageEvent.man
@@ -153,10 +153,10 @@ events contains:
 .TA .5i 1i 3i
 .ta .5i 1i 3i
 typedef struct {
-	int type;	/\(** ClientMessage */
-	unsigned long serial;		/\(** # of last request processed by server */
-	Bool send_event;		/\(** true if this came from a SendEvent request */
-	Display *display;		/\(** Display the event was read from */
+	int type;	/\&* ClientMessage */
+	unsigned long serial;		/\&* # of last request processed by server */
+	Bool send_event;		/\&* true if this came from a SendEvent request */
+	Display *display;		/\&* Display the event was read from */
 	Window window;
 	Atom message_type;
 	int format;
diff --git a/man/XColormapEvent.man b/man/XColormapEvent.man
index 95de9c9..8920c7f 100644
--- a/man/XColormapEvent.man
+++ b/man/XColormapEvent.man
@@ -153,14 +153,14 @@ events contains:
 .TA .5i 3i
 .ta .5i 3i
 typedef struct {
-	int type;	/\(** ColormapNotify */
-	unsigned long serial;	/\(** # of last request processed by server */
-	Bool send_event;	/\(** true if this came from a SendEvent request */
-	Display *display;	/\(** Display the event was read from */
+	int type;	/\&* ColormapNotify */
+	unsigned long serial;	/\&* # of last request processed by server */
+	Bool send_event;	/\&* true if this came from a SendEvent request */
+	Display *display;	/\&* Display the event was read from */
 	Window window;
-	Colormap colormap;	/\(** colormap or None */
+	Colormap colormap;	/\&* colormap or None */
 	Bool new;
-	int state;	/\(** ColormapInstalled, ColormapUninstalled */
+	int state;	/\&* ColormapInstalled, ColormapUninstalled */
 } XColormapEvent;
 .De
 .LP
diff --git a/man/XConfigureEvent.man b/man/XConfigureEvent.man
index 5cc65cf..47c8ea4 100644
--- a/man/XConfigureEvent.man
+++ b/man/XConfigureEvent.man
@@ -153,10 +153,10 @@ events contains:
 .TA .5i 3i
 .ta .5i 3i
 typedef struct {
-	int type;	/\(** ConfigureNotify */
-	unsigned long serial;	/\(** # of last request processed by server */
-	Bool send_event;	/\(** true if this came from a SendEvent request */
-	Display *display;	/\(** Display the event was read from */
+	int type;	/\&* ConfigureNotify */
+	unsigned long serial;	/\&* # of last request processed by server */
+	Bool send_event;	/\&* true if this came from a SendEvent request */
+	Display *display;	/\&* Display the event was read from */
 	Window event;
 	Window window;
 	int x, y;
diff --git a/man/XConfigureRequestEvent.man b/man/XConfigureRequestEvent.man
index dbf6c99..07c6709 100644
--- a/man/XConfigureRequestEvent.man
+++ b/man/XConfigureRequestEvent.man
@@ -153,17 +153,17 @@ events contains:
 .TA .5i 3i
 .ta .5i 3i
 typedef struct {
-	int type;	/\(** ConfigureRequest */
-	unsigned long serial;	/\(** # of last request processed by server */
-	Bool send_event;	/\(** true if this came from a SendEvent request */
-	Display *display;	/\(** Display the event was read from */
+	int type;	/\&* ConfigureRequest */
+	unsigned long serial;	/\&* # of last request processed by server */
+	Bool send_event;	/\&* true if this came from a SendEvent request */
+	Display *display;	/\&* Display the event was read from */
 	Window parent;
 	Window window;
 	int x, y;
 	int width, height;
 	int border_width;
 	Window above;
-	int detail;	/\(** Above, Below, TopIf, BottomIf, Opposite */
+	int detail;	/\&* Above, Below, TopIf, BottomIf, Opposite */
 	unsigned long value_mask;
 } XConfigureRequestEvent;
 .De
diff --git a/man/XConfigureWindow.man b/man/XConfigureWindow.man
index 54059c8..2541942 100644
--- a/man/XConfigureWindow.man
+++ b/man/XConfigureWindow.man
@@ -334,7 +334,7 @@ The
 structure contains:
 .LP
 .LP
-/\(** Configure window value mask bits */
+/\&* Configure window value mask bits */
 .TS
 lw(.5i) lw(2.5i) lw(.8i).
 T{
@@ -391,7 +391,7 @@ T}
 .Ds 0
 .TA .5i 3i
 .ta .5i 3i
-/\(** Values */
+/\&* Values */
 
 typedef struct {
 	int x, y;
diff --git a/man/XCreateColormap.man b/man/XCreateColormap.man
index 17c6dbf..0901775 100644
--- a/man/XCreateColormap.man
+++ b/man/XCreateColormap.man
@@ -313,9 +313,9 @@ structure contains:
 .TA .5i 2.5i
 .ta .5i 2.5i
 typedef struct {
-	unsigned long pixel;	/\(** pixel value */
-	unsigned short red, green, blue;	/\(** rgb values */
-	char flags;	/\(** DoRed, DoGreen, DoBlue */	
+	unsigned long pixel;	/\&* pixel value */
+	unsigned short red, green, blue;	/\&* rgb values */
+	char flags;	/\&* DoRed, DoGreen, DoBlue */	
 	char pad;
 } XColor;
 .De
diff --git a/man/XCreateGC.man b/man/XCreateGC.man
index 91897a0..9b698e6 100644
--- a/man/XCreateGC.man
+++ b/man/XCreateGC.man
@@ -311,7 +311,7 @@ The
 structure contains:
 .LP
 .LP
-/\(** GC attribute value mask bits */
+/\&* GC attribute value mask bits */
 .TS
 lw(.5i) lw(2.5i) lw(.75i).
 T{
@@ -480,31 +480,31 @@ T}
 .Ds 0
 .TA .5i 3i
 .ta .5i 3i
-/\(** Values */
+/\&* Values */
 
 typedef struct {
-	int function;	/\(** logical operation */
-	unsigned long plane_mask;	/\(** plane mask */
-	unsigned long foreground;	/\(** foreground pixel */
-	unsigned long background;	/\(** background pixel */
-	int line_width;	/\(** line width (in pixels) */
-	int line_style;	/\(** LineSolid, LineOnOffDash, LineDoubleDash */
-	int cap_style;	/\(** CapNotLast, CapButt, CapRound, CapProjecting */
-	int join_style;	/\(** JoinMiter, JoinRound, JoinBevel */
-	int fill_style;	/\(** FillSolid, FillTiled, FillStippled FillOpaqueStippled*/
-	int fill_rule;	/\(** EvenOddRule, WindingRule */
-	int arc_mode;	/\(** ArcChord, ArcPieSlice */
-	Pixmap tile;	/\(** tile pixmap for tiling operations */
-	Pixmap stipple;	/\(** stipple 1 plane pixmap for stippling */
-	int ts_x_origin;	/\(** offset for tile or stipple operations */
+	int function;	/\&* logical operation */
+	unsigned long plane_mask;	/\&* plane mask */
+	unsigned long foreground;	/\&* foreground pixel */
+	unsigned long background;	/\&* background pixel */
+	int line_width;	/\&* line width (in pixels) */
+	int line_style;	/\&* LineSolid, LineOnOffDash, LineDoubleDash */
+	int cap_style;	/\&* CapNotLast, CapButt, CapRound, CapProjecting */
+	int join_style;	/\&* JoinMiter, JoinRound, JoinBevel */
+	int fill_style;	/\&* FillSolid, FillTiled, FillStippled FillOpaqueStippled*/
+	int fill_rule;	/\&* EvenOddRule, WindingRule */
+	int arc_mode;	/\&* ArcChord, ArcPieSlice */
+	Pixmap tile;	/\&* tile pixmap for tiling operations */
+	Pixmap stipple;	/\&* stipple 1 plane pixmap for stippling */
+	int ts_x_origin;	/\&* offset for tile or stipple operations */
 	int ts_y_origin;
-	Font font;	/\(** default text font for text operations */
-	int subwindow_mode;	/\(** ClipByChildren, IncludeInferiors */
-	Bool graphics_exposures;	/\(** boolean, should exposures be generated */
-	int clip_x_origin;	/\(** origin for clipping */
+	Font font;	/\&* default text font for text operations */
+	int subwindow_mode;	/\&* ClipByChildren, IncludeInferiors */
+	Bool graphics_exposures;	/\&* boolean, should exposures be generated */
+	int clip_x_origin;	/\&* origin for clipping */
 	int clip_y_origin;
-	Pixmap clip_mask;	/\(** bitmap clipping; other calls for rects */
-	int dash_offset;	/\(** patterned/dashed line information */
+	Pixmap clip_mask;	/\&* bitmap clipping; other calls for rects */
+	int dash_offset;	/\&* patterned/dashed line information */
 	char dashes;
 } XGCValues;
 .De
diff --git a/man/XCreateWindow.man b/man/XCreateWindow.man
index 18c66df..c8e03ea 100644
--- a/man/XCreateWindow.man
+++ b/man/XCreateWindow.man
@@ -319,7 +319,7 @@ The
 structure contains:
 .LP
 .LP
-/\(** Window attribute value mask bits */
+/\&* Window attribute value mask bits */
 .TS
 lw(.5i) lw(2.5i) lw(.8i).
 T{
@@ -432,24 +432,24 @@ T}
 .Ds 0
 .TA .5i 3i
 .ta .5i 3i
-/\(** Values */
+/\&* Values */
 
 typedef struct {
-	Pixmap background_pixmap;	/\(** background, None, or ParentRelative */
-	unsigned long background_pixel;	/\(** background pixel */
-	Pixmap border_pixmap;		/\(** border of the window or CopyFromParent */
-	unsigned long border_pixel;	/\(** border pixel value */
-	int bit_gravity;	/\(** one of bit gravity values */
-	int win_gravity;	/\(** one of the window gravity values */
-	int backing_store;	/\(** NotUseful, WhenMapped, Always */
-	unsigned long backing_planes;	/\(** planes to be preserved if possible */
-	unsigned long backing_pixel;	/\(** value to use in restoring planes */
-	Bool save_under;	/\(** should bits under be saved? (popups) */
-	long event_mask;	/\(** set of events that should be saved */
-	long do_not_propagate_mask;	/\(** set of events that should not propagate */
-	Bool override_redirect;	/\(** boolean value for override_redirect */
-	Colormap colormap;	/\(** color map to be associated with window */
-	Cursor cursor;		/\(** cursor to be displayed (or None) */
+	Pixmap background_pixmap;	/\&* background, None, or ParentRelative */
+	unsigned long background_pixel;	/\&* background pixel */
+	Pixmap border_pixmap;		/\&* border of the window or CopyFromParent */
+	unsigned long border_pixel;	/\&* border pixel value */
+	int bit_gravity;	/\&* one of bit gravity values */
+	int win_gravity;	/\&* one of the window gravity values */
+	int backing_store;	/\&* NotUseful, WhenMapped, Always */
+	unsigned long backing_planes;	/\&* planes to be preserved if possible */
+	unsigned long backing_pixel;	/\&* value to use in restoring planes */
+	Bool save_under;	/\&* should bits under be saved? (popups) */
+	long event_mask;	/\&* set of events that should be saved */
+	long do_not_propagate_mask;	/\&* set of events that should not propagate */
+	Bool override_redirect;	/\&* boolean value for override_redirect */
+	Colormap colormap;	/\&* color map to be associated with window */
+	Cursor cursor;		/\&* cursor to be displayed (or None) */
 } XSetWindowAttributes;
 .De
 .LP
diff --git a/man/XCreateWindowEvent.man b/man/XCreateWindowEvent.man
index 9fb2ce2..8574f4e 100644
--- a/man/XCreateWindowEvent.man
+++ b/man/XCreateWindowEvent.man
@@ -153,16 +153,16 @@ events contains:
 .TA .5i 3i
 .ta .5i 3i
 typedef struct {
-	int type;	/\(** CreateNotify */
-	unsigned long serial;	/\(** # of last request processed by server */
-	Bool send_event;	/\(** true if this came from a SendEvent request */
-	Display *display;	/\(** Display the event was read from */
-	Window parent;	/\(** parent of the window */
-	Window window;	/\(** window id of window created */
-	int x, y;	/\(** window location */
-	int width, height;	/\(** size of window */
-	int border_width;	/\(** border width */
-	Bool override_redirect;	/\(** creation should be overridden */
+	int type;	/\&* CreateNotify */
+	unsigned long serial;	/\&* # of last request processed by server */
+	Bool send_event;	/\&* true if this came from a SendEvent request */
+	Display *display;	/\&* Display the event was read from */
+	Window parent;	/\&* parent of the window */
+	Window window;	/\&* window id of window created */
+	int x, y;	/\&* window location */
+	int width, height;	/\&* size of window */
+	int border_width;	/\&* border width */
+	Bool override_redirect;	/\&* creation should be overridden */
 } XCreateWindowEvent;
 .De
 .LP
diff --git a/man/XCrossingEvent.man b/man/XCrossingEvent.man
index 9721a2c..1556b48 100644
--- a/man/XCrossingEvent.man
+++ b/man/XCrossingEvent.man
@@ -155,25 +155,25 @@ events contains:
 .TA .5i 3i
 .ta .5i 3i
 typedef struct {
-	int type;	/\(** EnterNotify or LeaveNotify */
-	unsigned long serial;	/\(** # of last request processed by server */
-	Bool send_event;	/\(** true if this came from a SendEvent request */
-	Display *display;	/\(** Display the event was read from */
-	Window window;	/\(** ``event'' window reported relative to */
-	Window root;	/\(** root window that the event occurred on */
-	Window subwindow;	/\(** child window */
-	Time time;	/\(** milliseconds */
-	int x, y;	/\(** pointer x, y coordinates in event window */
-	int x_root, y_root;	/\(** coordinates relative to root */
-	int mode;	/\(** NotifyNormal, NotifyGrab, NotifyUngrab */
+	int type;	/\&* EnterNotify or LeaveNotify */
+	unsigned long serial;	/\&* # of last request processed by server */
+	Bool send_event;	/\&* true if this came from a SendEvent request */
+	Display *display;	/\&* Display the event was read from */
+	Window window;	/\&* ``event'' window reported relative to */
+	Window root;	/\&* root window that the event occurred on */
+	Window subwindow;	/\&* child window */
+	Time time;	/\&* milliseconds */
+	int x, y;	/\&* pointer x, y coordinates in event window */
+	int x_root, y_root;	/\&* coordinates relative to root */
+	int mode;	/\&* NotifyNormal, NotifyGrab, NotifyUngrab */
 	int detail;
-		/\(**
+		/\&*
 	 	* NotifyAncestor, NotifyVirtual, NotifyInferior, 
 	 	* NotifyNonlinear,NotifyNonlinearVirtual
 	 	*/
-	Bool same_screen;	/\(** same screen flag */
-	Bool focus;	/\(** boolean focus */
-	unsigned int state;	/\(** key or button mask */
+	Bool same_screen;	/\&* same screen flag */
+	Bool focus;	/\&* boolean focus */
+	unsigned int state;	/\&* key or button mask */
 } XCrossingEvent;
 typedef XCrossingEvent XEnterWindowEvent;
 typedef XCrossingEvent XLeaveWindowEvent;
diff --git a/man/XDestroyWindowEvent.man b/man/XDestroyWindowEvent.man
index 9c5a261..91c024b 100644
--- a/man/XDestroyWindowEvent.man
+++ b/man/XDestroyWindowEvent.man
@@ -153,10 +153,10 @@ events contains:
 .TA .5i 3i
 .ta .5i 3i
 typedef struct {
-	int type;	/\(** DestroyNotify */
-	unsigned long serial;	/\(** # of last request processed by server */
-	Bool send_event;	/\(** true if this came from a SendEvent request */
-	Display *display;	/\(** Display the event was read from */
+	int type;	/\&* DestroyNotify */
+	unsigned long serial;	/\&* # of last request processed by server */
+	Bool send_event;	/\&* true if this came from a SendEvent request */
+	Display *display;	/\&* Display the event was read from */
 	Window event;
 	Window window;
 } XDestroyWindowEvent;
diff --git a/man/XDrawArc.man b/man/XDrawArc.man
index 00a79cf..af25276 100644
--- a/man/XDrawArc.man
+++ b/man/XDrawArc.man
@@ -308,7 +308,7 @@ structure contains:
 typedef struct {
 	short x, y;
 	unsigned short width, height;
-	short angle1, angle2;             /\(** Degrees * 64 */
+	short angle1, angle2;             /\&* Degrees * 64 */
 } XArc;
 .EE
 .LP
diff --git a/man/XDrawText.man b/man/XDrawText.man
index 706d69e..ae2e44e 100644
--- a/man/XDrawText.man
+++ b/man/XDrawText.man
@@ -228,10 +228,10 @@ structures contain:
 .TA .5i 3i
 .ta .5i 3i
 typedef struct {
-	char *chars;	/\(** pointer to string */
-	int nchars;	/\(** number of characters */
-	int delta;	/\(** delta between strings */
-	Font font;	/\(** Font to print it in, None don't change */
+	char *chars;	/\&* pointer to string */
+	int nchars;	/\&* number of characters */
+	int delta;	/\&* delta between strings */
+	Font font;	/\&* Font to print it in, None don't change */
 } XTextItem;
 .De
 .LP
@@ -239,10 +239,10 @@ typedef struct {
 .TA .5i 3i
 .ta .5i 3i
 typedef struct {
-	XChar2b *chars;	/\(** pointer to two-byte characters */
-	int nchars;	/\(** number of characters */
-	int delta;	/\(** delta between strings */
-	Font font;	/\(** font to print it in, None don't change */
+	XChar2b *chars;	/\&* pointer to two-byte characters */
+	int nchars;	/\&* number of characters */
+	int delta;	/\&* delta between strings */
+	Font font;	/\&* font to print it in, None don't change */
 } XTextItem16;
 .De
 .LP
diff --git a/man/XErrorEvent.man b/man/XErrorEvent.man
index e1f4eae..c74b46b 100644
--- a/man/XErrorEvent.man
+++ b/man/XErrorEvent.man
@@ -154,12 +154,12 @@ structure contains:
 .ta .5i 2.5i
 typedef struct {
 	int type;
-	Display *display;	/\(** Display the event was read from */
-	unsigned long serial;		/\(** serial number of failed request */
-	unsigned char error_code;	/\(** error code of failed request */
-	unsigned char request_code;	/\(** Major op-code of failed request */
-	unsigned char minor_code;	/\(** Minor op-code of failed request */
-	XID resourceid;		/\(** resource id */
+	Display *display;	/\&* Display the event was read from */
+	unsigned long serial;		/\&* serial number of failed request */
+	unsigned char error_code;	/\&* error code of failed request */
+	unsigned char request_code;	/\&* Major op-code of failed request */
+	unsigned char minor_code;	/\&* Minor op-code of failed request */
+	XID resourceid;		/\&* resource id */
 } XErrorEvent;
 .De
 .LP
diff --git a/man/XExposeEvent.man b/man/XExposeEvent.man
index 82efac2..84cf712 100644
--- a/man/XExposeEvent.man
+++ b/man/XExposeEvent.man
@@ -153,14 +153,14 @@ events contains:
 .TA .5i 3i
 .ta .5i 3i
 typedef struct {
-	int type;	/\(** Expose */
-	unsigned long serial;	/\(** # of last request processed by server */
-	Bool send_event;	/\(** true if this came from a SendEvent request */
-	Display *display;	/\(** Display the event was read from */
+	int type;	/\&* Expose */
+	unsigned long serial;	/\&* # of last request processed by server */
+	Bool send_event;	/\&* true if this came from a SendEvent request */
+	Display *display;	/\&* Display the event was read from */
 	Window window;
 	int x, y;
 	int width, height;
-	int count;	/\(** if nonzero, at least this many more */
+	int count;	/\&* if nonzero, at least this many more */
 } XExposeEvent;
 .De
 .LP
diff --git a/man/XFocusChangeEvent.man b/man/XFocusChangeEvent.man
index c96d3cc..61f38a7 100644
--- a/man/XFocusChangeEvent.man
+++ b/man/XFocusChangeEvent.man
@@ -155,14 +155,14 @@ events contains:
 .TA .5i 3i
 .ta .5i 3i
 typedef struct {
-	int type;	/\(** FocusIn or FocusOut */
-	unsigned long serial;	/\(** # of last request processed by server */
-	Bool send_event;	/\(** true if this came from a SendEvent request */
-	Display *display;	/\(** Display the event was read from */
-	Window window;	/\(** window of event */
-	int mode;	/\(** NotifyNormal, NotifyGrab, NotifyUngrab */
+	int type;	/\&* FocusIn or FocusOut */
+	unsigned long serial;	/\&* # of last request processed by server */
+	Bool send_event;	/\&* true if this came from a SendEvent request */
+	Display *display;	/\&* Display the event was read from */
+	Window window;	/\&* window of event */
+	int mode;	/\&* NotifyNormal, NotifyGrab, NotifyUngrab */
 	int detail;
-		/\(**
+		/\&*
 	 	* NotifyAncestor, NotifyVirtual, NotifyInferior, 
 	 	* NotifyNonlinear,NotifyNonlinearVirtual, NotifyPointer,
 	 	* NotifyPointerRoot, NotifyDetailNone 
diff --git a/man/XFontSetExtents.man b/man/XFontSetExtents.man
index 6d415ba..d70428f 100644
--- a/man/XFontSetExtents.man
+++ b/man/XFontSetExtents.man
@@ -153,8 +153,8 @@ structure contains:
 .TA .5i 3i
 .ta .5i 3i
 typedef struct {
-	XRectangle max_ink_extent;	/\(** over all drawable characters */
-	XRectangle max_logical_extent;	/\(** over all drawable characters */
+	XRectangle max_ink_extent;	/\&* over all drawable characters */
+	XRectangle max_logical_extent;	/\&* over all drawable characters */
 } XFontSetExtents;
 .De
 .LP
diff --git a/man/XGetVisualInfo.man b/man/XGetVisualInfo.man
index 155c9f1..e4f7937 100644
--- a/man/XGetVisualInfo.man
+++ b/man/XGetVisualInfo.man
@@ -209,7 +209,7 @@ The
 structure contains:
 .LP
 .LP
-/\(** Visual information mask bits */
+/\&* Visual information mask bits */
 .TS
 lw(.5i) lw(2.5i) lw(.8i).
 T{
@@ -294,7 +294,7 @@ T}
 .Ds 0
 .TA .5i 3i
 .ta .5i 3i
-/\(** Values */
+/\&* Values */
 
 typedef struct {
 	Visual *visual;
diff --git a/man/XGetWindowAttributes.man b/man/XGetWindowAttributes.man
index b7665b7..2aa86e4 100644
--- a/man/XGetWindowAttributes.man
+++ b/man/XGetWindowAttributes.man
@@ -222,27 +222,27 @@ structure contains:
 .TA .5i 3i
 .ta .5i 3i
 typedef struct {
-	int x, y;	/\(** location of window */
-	int width, height;	/\(** width and height of window */
-	int border_width;	/\(** border width of window */
-	int depth;	/\(** depth of window */
-	Visual *visual;	/\(** the associated visual structure */
-	Window root;	/\(** root of screen containing window */
-	int class;	/\(** InputOutput, InputOnly*/
-	int bit_gravity;	/\(** one of the bit gravity values */
-	int win_gravity;	/\(** one of the window gravity values */
-	int backing_store;	/\(** NotUseful, WhenMapped, Always */
-	unsigned long backing_planes;	/\(** planes to be preserved if possible */
-	unsigned long backing_pixel;	/\(** value to be used when restoring planes */
-	Bool save_under;	/\(** boolean, should bits under be saved? */
-	Colormap colormap;	/\(** color map to be associated with window */
-	Bool map_installed;	/\(** boolean, is color map currently installed*/
-	int map_state;	/\(** IsUnmapped, IsUnviewable, IsViewable */
-	long all_event_masks;	/\(** set of events all people have interest in*/
-	long your_event_mask;	/\(** my event mask */
-	long do_not_propagate_mask;	/\(** set of events that should not propagate */
-	Bool override_redirect;	/\(** boolean value for override-redirect */
-	Screen *screen;	/\(** back pointer to correct screen */
+	int x, y;	/\&* location of window */
+	int width, height;	/\&* width and height of window */
+	int border_width;	/\&* border width of window */
+	int depth;	/\&* depth of window */
+	Visual *visual;	/\&* the associated visual structure */
+	Window root;	/\&* root of screen containing window */
+	int class;	/\&* InputOutput, InputOnly*/
+	int bit_gravity;	/\&* one of the bit gravity values */
+	int win_gravity;	/\&* one of the window gravity values */
+	int backing_store;	/\&* NotUseful, WhenMapped, Always */
+	unsigned long backing_planes;	/\&* planes to be preserved if possible */
+	unsigned long backing_pixel;	/\&* value to be used when restoring planes */
+	Bool save_under;	/\&* boolean, should bits under be saved? */
+	Colormap colormap;	/\&* color map to be associated with window */
+	Bool map_installed;	/\&* boolean, is color map currently installed*/
+	int map_state;	/\&* IsUnmapped, IsUnviewable, IsViewable */
+	long all_event_masks;	/\&* set of events all people have interest in*/
+	long your_event_mask;	/\&* my event mask */
+	long do_not_propagate_mask;	/\&* set of events that should not propagate */
+	Bool override_redirect;	/\&* boolean value for override-redirect */
+	Screen *screen;	/\&* back pointer to correct screen */
 } XWindowAttributes;
 .De
 .LP
diff --git a/man/XGraphicsExposeEvent.man b/man/XGraphicsExposeEvent.man
index 6e20083..a5637c8 100644
--- a/man/XGraphicsExposeEvent.man
+++ b/man/XGraphicsExposeEvent.man
@@ -155,16 +155,16 @@ events contain:
 .TA .5i 3i
 .ta .5i 3i
 typedef struct {
-	int type;	/\(** GraphicsExpose */
-	unsigned long serial;	/\(** # of last request processed by server */
-	Bool send_event;	/\(** true if this came from a SendEvent request */
-	Display *display;	/\(** Display the event was read from */
+	int type;	/\&* GraphicsExpose */
+	unsigned long serial;	/\&* # of last request processed by server */
+	Bool send_event;	/\&* true if this came from a SendEvent request */
+	Display *display;	/\&* Display the event was read from */
 	Drawable drawable;
 	int x, y;
 	int width, height;
-	int count;	/\(** if nonzero, at least this many more */
-	int major_code;	/\(** core is CopyArea or CopyPlane */
-	int minor_code;	/\(** not defined in the core */
+	int count;	/\&* if nonzero, at least this many more */
+	int major_code;	/\&* core is CopyArea or CopyPlane */
+	int minor_code;	/\&* not defined in the core */
 } XGraphicsExposeEvent;
 .De
 .LP
@@ -172,13 +172,13 @@ typedef struct {
 .TA .5i 3i
 .ta .5i 3i
 typedef struct {
-	int type;	/\(** NoExpose */
-	unsigned long serial;	/\(** # of last request processed by server */
-	Bool send_event;	/\(** true if this came from a SendEvent request */
-	Display *display;	/\(** Display the event was read from */
+	int type;	/\&* NoExpose */
+	unsigned long serial;	/\&* # of last request processed by server */
+	Bool send_event;	/\&* true if this came from a SendEvent request */
+	Display *display;	/\&* Display the event was read from */
 	Drawable drawable;
-	int major_code;	/\(** core is CopyArea or CopyPlane */
-	int minor_code;	/\(** not defined in the core */
+	int major_code;	/\&* core is CopyArea or CopyPlane */
+	int minor_code;	/\&* not defined in the core */
 } XNoExposeEvent;
 .De
 .LP
diff --git a/man/XGravityEvent.man b/man/XGravityEvent.man
index 9d07e6f..c49719e 100644
--- a/man/XGravityEvent.man
+++ b/man/XGravityEvent.man
@@ -153,10 +153,10 @@ events contains:
 .TA .5i 3i
 .ta .5i 3i
 typedef struct {
-	int type;	/\(** GravityNotify */
-	unsigned long serial;	/\(** # of last request processed by server */
-	Bool send_event;	/\(** true if this came from a SendEvent request */
-	Display *display;	/\(** Display the event was read from */
+	int type;	/\&* GravityNotify */
+	unsigned long serial;	/\&* # of last request processed by server */
+	Bool send_event;	/\&* true if this came from a SendEvent request */
+	Display *display;	/\&* Display the event was read from */
 	Window event;
 	Window window;
 	int x, y;
diff --git a/man/XKeymapEvent.man b/man/XKeymapEvent.man
index c687c96..356865d 100644
--- a/man/XKeymapEvent.man
+++ b/man/XKeymapEvent.man
@@ -152,12 +152,12 @@ events contains:
 .Ds 0
 .TA .5i 3i
 .ta .5i 3i
-/\(** generated on EnterWindow and FocusIn when KeymapState selected */
+/\&* generated on EnterWindow and FocusIn when KeymapState selected */
 typedef struct {
-	int type;	/\(** KeymapNotify */
-	unsigned long serial;	/\(** # of last request processed by server */
-	Bool send_event;	/\(** true if this came from a SendEvent request */
-	Display *display;	/\(** Display the event was read from */
+	int type;	/\&* KeymapNotify */
+	unsigned long serial;	/\&* # of last request processed by server */
+	Bool send_event;	/\&* true if this came from a SendEvent request */
+	Display *display;	/\&* Display the event was read from */
 	Window window;
 	char key_vector[32];
 } XKeymapEvent;	
diff --git a/man/XLoadFont.man b/man/XLoadFont.man
index 906f20e..a13ed84 100644
--- a/man/XLoadFont.man
+++ b/man/XLoadFont.man
@@ -321,12 +321,12 @@ structures contain:
 .IN "XCharStruct" "" "@DEF@"
 .EX
 typedef struct {
-	short lbearing;	/\(** origin to left edge of raster */
-	short rbearing;	/\(** origin to right edge of raster */
-	short width;	/\(** advance to next char's origin */
-	short ascent;	/\(** baseline to top edge of raster */
-	short descent;	/\(** baseline to bottom edge of raster */
-	unsigned short attributes;	/\(** per char flags (not predefined) */
+	short lbearing;	/\&* origin to left edge of raster */
+	short rbearing;	/\&* origin to right edge of raster */
+	short width;	/\&* advance to next char's origin */
+	short ascent;	/\&* baseline to top edge of raster */
+	short descent;	/\&* baseline to bottom edge of raster */
+	unsigned short attributes;	/\&* per char flags (not predefined) */
 } XCharStruct;
 .EE
 .LP
@@ -340,7 +340,7 @@ typedef struct {
 .LP
 .IN "XChar2b" "" "@DEF@"
 .EX
-typedef struct {	/\(** normal 16 bit characters are two bytes */
+typedef struct {	/\&* normal 16 bit characters are two bytes */
     unsigned char byte1;
     unsigned char byte2;
 } XChar2b;
@@ -349,22 +349,22 @@ typedef struct {	/\(** normal 16 bit cha
 .IN "XFontStruct" "" "@DEF@"
 .EX
 typedef struct {
-	XExtData *ext_data;	/\(** hook for extension to hang data */
-	Font fid;	/\(** Font id for this font */
-	unsigned direction;	/\(** hint about the direction font is painted */
-	unsigned min_char_or_byte2;	/\(** first character */
-	unsigned max_char_or_byte2;	/\(** last character */
-	unsigned min_byte1;	/\(** first row that exists */
-	unsigned max_byte1;	/\(** last row that exists */
-	Bool all_chars_exist;	/\(** flag if all characters have nonzero size */
-	unsigned default_char;	/\(** char to print for undefined character */
-	int n_properties;	/\(** how many properties there are */
-	XFontProp *properties;	/\(** pointer to array of additional properties */
-	XCharStruct min_bounds;	/\(** minimum bounds over all existing char */
-	XCharStruct max_bounds;	/\(** maximum bounds over all existing char */
-	XCharStruct *per_char;	/\(** first_char to last_char information */
-	int ascent;	/\(** logical extent above baseline for spacing */
-	int descent;	/\(** logical decent below baseline for spacing */
+	XExtData *ext_data;	/\&* hook for extension to hang data */
+	Font fid;	/\&* Font id for this font */
+	unsigned direction;	/\&* hint about the direction font is painted */
+	unsigned min_char_or_byte2;	/\&* first character */
+	unsigned max_char_or_byte2;	/\&* last character */
+	unsigned min_byte1;	/\&* first row that exists */
+	unsigned max_byte1;	/\&* last row that exists */
+	Bool all_chars_exist;	/\&* flag if all characters have nonzero size */
+	unsigned default_char;	/\&* char to print for undefined character */
+	int n_properties;	/\&* how many properties there are */
+	XFontProp *properties;	/\&* pointer to array of additional properties */
+	XCharStruct min_bounds;	/\&* minimum bounds over all existing char */
+	XCharStruct max_bounds;	/\&* maximum bounds over all existing char */
+	XCharStruct *per_char;	/\&* first_char to last_char information */
+	int ascent;	/\&* logical extent above baseline for spacing */
+	int descent;	/\&* logical decent below baseline for spacing */
 } XFontStruct;
 .EE
 .LP
diff --git a/man/XMapEvent.man b/man/XMapEvent.man
index 5bc8f85..7f2747c 100644
--- a/man/XMapEvent.man
+++ b/man/XMapEvent.man
@@ -153,13 +153,13 @@ events contains:
 .TA .5i 3i
 .ta .5i 3i
 typedef struct {
-	int type;	/\(** MapNotify */
-	unsigned long serial;	/\(** # of last request processed by server */
-	Bool send_event;	/\(** true if this came from a SendEvent request */
-	Display *display;	/\(** Display the event was read from */
+	int type;	/\&* MapNotify */
+	unsigned long serial;	/\&* # of last request processed by server */
+	Bool send_event;	/\&* true if this came from a SendEvent request */
+	Display *display;	/\&* Display the event was read from */
 	Window event;
 	Window window;
-	Bool override_redirect;	/\(** boolean, is override set... */
+	Bool override_redirect;	/\&* boolean, is override set... */
 } XMapEvent;
 .De
 .LP
@@ -208,15 +208,15 @@ events is:
 .TA .5i 3i
 .ta .5i 3i
 typedef struct {
-	int type;	/\(** MappingNotify */
-	unsigned long serial;	/\(** # of last request processed by server */
-	Bool send_event;	/\(** true if this came from a SendEvent request */
-	Display *display;	/\(** Display the event was read from */
-	Window window;	/\(** unused */
-	int request;	/\(** one of MappingModifier, MappingKeyboard,
+	int type;	/\&* MappingNotify */
+	unsigned long serial;	/\&* # of last request processed by server */
+	Bool send_event;	/\&* true if this came from a SendEvent request */
+	Display *display;	/\&* Display the event was read from */
+	Window window;	/\&* unused */
+	int request;	/\&* one of MappingModifier, MappingKeyboard,
 		   MappingPointer */
-	int first_keycode;	/\(** first keycode */
-	int count;	/\(** defines range of change w. first_keycode*/
+	int first_keycode;	/\&* first keycode */
+	int count;	/\&* defines range of change w. first_keycode*/
 } XMappingEvent;
 .De
 .LP
diff --git a/man/XMapRequestEvent.man b/man/XMapRequestEvent.man
index f76551d..a8ece72 100644
--- a/man/XMapRequestEvent.man
+++ b/man/XMapRequestEvent.man
@@ -153,10 +153,10 @@ events contains:
 .TA .5i 3i
 .ta .5i 3i
 typedef struct {
-	int type;	/\(** MapRequest */
-	unsigned long serial;	/\(** # of last request processed by server */
-	Bool send_event;	/\(** true if this came from a SendEvent request */
-	Display *display;	/\(** Display the event was read from */
+	int type;	/\&* MapRequest */
+	unsigned long serial;	/\&* # of last request processed by server */
+	Bool send_event;	/\&* true if this came from a SendEvent request */
+	Display *display;	/\&* Display the event was read from */
 	Window parent;
 	Window window;
 } XMapRequestEvent;
diff --git a/man/XPropertyEvent.man b/man/XPropertyEvent.man
index ef34642..30ac75a 100644
--- a/man/XPropertyEvent.man
+++ b/man/XPropertyEvent.man
@@ -153,14 +153,14 @@ events contains:
 .TA .5i 3i
 .ta .5i 3i
 typedef struct {
-	int type;	/\(** PropertyNotify */
-	unsigned long serial;	/\(** # of last request processed by server */
-	Bool send_event;	/\(** true if this came from a SendEvent request */
-	Display *display;	/\(** Display the event was read from */
+	int type;	/\&* PropertyNotify */
+	unsigned long serial;	/\&* # of last request processed by server */
+	Bool send_event;	/\&* true if this came from a SendEvent request */
+	Display *display;	/\&* Display the event was read from */
 	Window window;
 	Atom atom;
 	Time time;
-	int state;	/\(** PropertyNewValue or PropertyDelete */
+	int state;	/\&* PropertyNewValue or PropertyDelete */
 } XPropertyEvent;
 .De
 .LP
diff --git a/man/XReparentEvent.man b/man/XReparentEvent.man
index ff42e28..228661c 100644
--- a/man/XReparentEvent.man
+++ b/man/XReparentEvent.man
@@ -153,10 +153,10 @@ events contains:
 .TA .5i 3i
 .ta .5i 3i
 typedef struct {
-	int type;	/\(** ReparentNotify */
-	unsigned long serial;	/\(** # of last request processed by server */
-	Bool send_event;	/\(** true if this came from a SendEvent request */
-	Display *display;	/\(** Display the event was read from */
+	int type;	/\&* ReparentNotify */
+	unsigned long serial;	/\&* # of last request processed by server */
+	Bool send_event;	/\&* true if this came from a SendEvent request */
+	Display *display;	/\&* Display the event was read from */
 	Window event;
 	Window window;
 	Window parent;
diff --git a/man/XResizeRequestEvent.man b/man/XResizeRequestEvent.man
index 71243b9..eb522f3 100644
--- a/man/XResizeRequestEvent.man
+++ b/man/XResizeRequestEvent.man
@@ -153,10 +153,10 @@ events contains:
 .TA .5i 3i
 .ta .5i 3i
 typedef struct {
-	int type;	/\(** ResizeRequest */
-	unsigned long serial;	/\(** # of last request processed by server */
-	Bool send_event;	/\(** true if this came from a SendEvent request */
-	Display *display;	/\(** Display the event was read from */
+	int type;	/\&* ResizeRequest */
+	unsigned long serial;	/\&* # of last request processed by server */
+	Bool send_event;	/\&* true if this came from a SendEvent request */
+	Display *display;	/\&* Display the event was read from */
 	Window window;
 	int width, height;
 } XResizeRequestEvent;
diff --git a/man/XSelectionClearEvent.man b/man/XSelectionClearEvent.man
index 3fd7b1f..d186034 100644
--- a/man/XSelectionClearEvent.man
+++ b/man/XSelectionClearEvent.man
@@ -153,10 +153,10 @@ events contains:
 .TA .5i 3i
 .ta .5i 3i
 typedef struct {
-	int type;	/\(** SelectionClear */
-	unsigned long serial;	/\(** # of last request processed by server */
-	Bool send_event;	/\(** true if this came from a SendEvent request */
-	Display *display;	/\(** Display the event was read from */
+	int type;	/\&* SelectionClear */
+	unsigned long serial;	/\&* # of last request processed by server */
+	Bool send_event;	/\&* true if this came from a SendEvent request */
+	Display *display;	/\&* Display the event was read from */
 	Window window;
 	Atom selection;
 	Time time;
diff --git a/man/XSelectionEvent.man b/man/XSelectionEvent.man
index 45481f5..e68eef7 100644
--- a/man/XSelectionEvent.man
+++ b/man/XSelectionEvent.man
@@ -153,14 +153,14 @@ events contains:
 .TA .5i 3i
 .ta .5i 3i
 typedef struct {
-	int type;	/\(** SelectionNotify */
-	unsigned long serial;	/\(** # of last request processed by server */
-	Bool send_event;	/\(** true if this came from a SendEvent request */
-	Display *display;	/\(** Display the event was read from */
+	int type;	/\&* SelectionNotify */
+	unsigned long serial;	/\&* # of last request processed by server */
+	Bool send_event;	/\&* true if this came from a SendEvent request */
+	Display *display;	/\&* Display the event was read from */
 	Window requestor;
 	Atom selection;
 	Atom target;
-	Atom property;	/\(** atom or None */
+	Atom property;	/\&* atom or None */
 	Time time;
 } XSelectionEvent;
 .De
diff --git a/man/XSelectionRequestEvent.man b/man/XSelectionRequestEvent.man
index 9d2febe..ae15700 100644
--- a/man/XSelectionRequestEvent.man
+++ b/man/XSelectionRequestEvent.man
@@ -153,10 +153,10 @@ events contains:
 .TA .5i 3i
 .ta .5i 3i
 typedef struct {
-	int type;	/\(** SelectionRequest */
-	unsigned long serial;	/\(** # of last request processed by server */
-	Bool send_event;	/\(** true if this came from a SendEvent request */
-	Display *display;	/\(** Display the event was read from */
+	int type;	/\&* SelectionRequest */
+	unsigned long serial;	/\&* # of last request processed by server */
+	Bool send_event;	/\&* true if this came from a SendEvent request */
+	Display *display;	/\&* Display the event was read from */
 	Window owner;
 	Window requestor;
 	Atom selection;
diff --git a/man/XStringListToTextProperty.man b/man/XStringListToTextProperty.man
index a921048..abba419 100644
--- a/man/XStringListToTextProperty.man
+++ b/man/XStringListToTextProperty.man
@@ -230,10 +230,10 @@ structure contains:
 .TA .5i 2.5i
 .ta .5i 2.5i
 typedef struct {
-	unsigned char *value;	/\(** property data */
-	Atom encoding;	/\(** type of property */
-	int format;	/\(** 8, 16, or 32 */
-	unsigned long nitems;	/\(** number of items in value */
+	unsigned char *value;	/\&* property data */
+	Atom encoding;	/\&* type of property */
+	int format;	/\&* 8, 16, or 32 */
+	unsigned long nitems;	/\&* number of items in value */
 } XTextProperty;
 .De
 .SH "SEE ALSO"
diff --git a/man/XUnmapEvent.man b/man/XUnmapEvent.man
index c560a2a..a0623f7 100644
--- a/man/XUnmapEvent.man
+++ b/man/XUnmapEvent.man
@@ -153,10 +153,10 @@ events contains:
 .TA .5i 3i
 .ta .5i 3i
 typedef struct {
-	int type;	/\(** UnmapNotify */
-	unsigned long serial;	/\(** # of last request processed by server */
-	Bool send_event;	/\(** true if this came from a SendEvent request */
-	Display *display;	/\(** Display the event was read from */
+	int type;	/\&* UnmapNotify */
+	unsigned long serial;	/\&* # of last request processed by server */
+	Bool send_event;	/\&* true if this came from a SendEvent request */
+	Display *display;	/\&* Display the event was read from */
 	Window event;
 	Window window;
 	Bool from_configure;
diff --git a/man/XVisibilityEvent.man b/man/XVisibilityEvent.man
index 0313aab..7b878cf 100644
--- a/man/XVisibilityEvent.man
+++ b/man/XVisibilityEvent.man
@@ -153,10 +153,10 @@ events contains:
 .TA .5i 3i
 .ta .5i 3i
 typedef struct {
-	int type;	/\(** VisibilityNotify */
-	unsigned long serial;	/\(** # of last request processed by server */
-	Bool send_event;	/\(** true if this came from a SendEvent request */
-	Display *display;	/\(** Display the event was read from */
+	int type;	/\&* VisibilityNotify */
+	unsigned long serial;	/\&* # of last request processed by server */
+	Bool send_event;	/\&* true if this came from a SendEvent request */
+	Display *display;	/\&* Display the event was read from */
 	Window window;
 	int state;
 } XVisibilityEvent;
diff --git a/man/XcmsColor.man b/man/XcmsColor.man
index 8c7d0e9..fa845dd 100644
--- a/man/XcmsColor.man
+++ b/man/XcmsColor.man
@@ -159,7 +159,7 @@ The structure for
 contains:
 .LP
 .EX
-typedef unsigned long XcmsColorFormat;			/\(** Color Specification Format */
+typedef unsigned long XcmsColorFormat;			/\&* Color Specification Format */
 
 typedef struct {
 	union {
@@ -175,73 +175,73 @@ typedef struct {
 	} spec;
 	unsigned long pixel;
 	XcmsColorFormat format;
-} XcmsColor;			/\(** Xcms Color Structure */
+} XcmsColor;			/\&* Xcms Color Structure */
 .EE
 .LP
 .EX
 typedef double XcmsFloat;
 
 typedef struct {
-	unsigned short red;	/\(** 0x0000 to 0xffff */
-	unsigned short green;	/\(** 0x0000 to 0xffff */
-	unsigned short blue;	/\(** 0x0000 to 0xffff */
-} XcmsRGB;		/\(** RGB Device */
+	unsigned short red;	/\&* 0x0000 to 0xffff */
+	unsigned short green;	/\&* 0x0000 to 0xffff */
+	unsigned short blue;	/\&* 0x0000 to 0xffff */
+} XcmsRGB;		/\&* RGB Device */
 .EE
 .LP
 .EX
 typedef struct {
-	XcmsFloat red;	/\(** 0.0 to 1.0 */
-	XcmsFloat green;	/\(** 0.0 to 1.0 */
-	XcmsFloat blue;	/\(** 0.0 to 1.0 */
-} XcmsRGBi;		/\(** RGB Intensity */
+	XcmsFloat red;	/\&* 0.0 to 1.0 */
+	XcmsFloat green;	/\&* 0.0 to 1.0 */
+	XcmsFloat blue;	/\&* 0.0 to 1.0 */
+} XcmsRGBi;		/\&* RGB Intensity */
 .EE
 .LP
 .EX
 typedef struct {
 	XcmsFloat X;
-	XcmsFloat Y;	/\(** 0.0 to 1.0 */
+	XcmsFloat Y;	/\&* 0.0 to 1.0 */
 	XcmsFloat Z;
-} XcmsCIEXYZ;		/\(** CIE XYZ */
+} XcmsCIEXYZ;		/\&* CIE XYZ */
 .EE
 .LP
 .EX
 typedef struct {
-	XcmsFloat u_prime;	/\(** 0.0 to ~0.6 */
-	XcmsFloat v_prime;	/\(** 0.0 to ~0.6 */
-	XcmsFloat Y; 	/\(** 0.0 to 1.0 */
-} XcmsCIEuvY;		/\(** CIE u'v'Y */
+	XcmsFloat u_prime;	/\&* 0.0 to ~0.6 */
+	XcmsFloat v_prime;	/\&* 0.0 to ~0.6 */
+	XcmsFloat Y; 	/\&* 0.0 to 1.0 */
+} XcmsCIEuvY;		/\&* CIE u'v'Y */
 .EE
 .LP
 .EX
 typedef struct {
-	XcmsFloat x; 	/\(** 0.0 to ~.75 */
-	XcmsFloat y; 	/\(** 0.0 to ~.85 */
-	XcmsFloat Y; 	/\(** 0.0 to 1.0 */
-} XcmsCIExyY;		/\(** CIE xyY */
+	XcmsFloat x; 	/\&* 0.0 to ~.75 */
+	XcmsFloat y; 	/\&* 0.0 to ~.85 */
+	XcmsFloat Y; 	/\&* 0.0 to 1.0 */
+} XcmsCIExyY;		/\&* CIE xyY */
 .EE
 .LP
 .EX
 typedef struct {
-	XcmsFloat L_star; 	/\(** 0.0 to 100.0 */
+	XcmsFloat L_star; 	/\&* 0.0 to 100.0 */
 	XcmsFloat a_star;
 	XcmsFloat b_star;
-} XcmsCIELab;		/\(** CIE L*a*b* */
+} XcmsCIELab;		/\&* CIE L*a*b* */
 .EE
 .LP
 .EX
 typedef struct {
-	XcmsFloat L_star; 	/\(** 0.0 to 100.0 */
+	XcmsFloat L_star; 	/\&* 0.0 to 100.0 */
 	XcmsFloat u_star;
 	XcmsFloat v_star;
-} XcmsCIELuv;		/\(** CIE L*u*v* */
+} XcmsCIELuv;		/\&* CIE L*u*v* */
 .EE
 .LP
 .EX
 typedef struct {
-	XcmsFloat H; 	/\(** 0.0 to 360.0 */
-	XcmsFloat V; 	/\(** 0.0 to 100.0 */
-	XcmsFloat C; 	/\(** 0.0 to 100.0 */
-} XcmsTekHVC;		/\(** TekHVC */
+	XcmsFloat H; 	/\&* 0.0 to 360.0 */
+	XcmsFloat V; 	/\&* 0.0 to 100.0 */
+	XcmsFloat C; 	/\&* 0.0 to 100.0 */
+} XcmsTekHVC;		/\&* TekHVC */
 .EE
 .LP
 .EX
@@ -250,7 +250,7 @@ typedef struct {
 	XcmsFloat pad1;
 	XcmsFloat pad2;
 	XcmsFloat pad3;
-} XcmsPad;		/\(** four doubles */
+} XcmsPad;		/\&* four doubles */
 .EE
 .SH DESCRIPTION
 The
diff --git a/man/XmbDrawText.man b/man/XmbDrawText.man
index 30fe5ea..b857f5a 100644
--- a/man/XmbDrawText.man
+++ b/man/XmbDrawText.man
@@ -228,10 +228,10 @@ structure contains:
 .TA .5i 2.5i
 .ta .5i 2.5i
 typedef struct {
-	char *chars;	/\(** pointer to string */
-	int nchars;	/\(** number of bytes */
-	int delta;	/\(** pixel delta between strings */
-	XFontSet font_set; 	/\(** fonts, None means don't change */
+	char *chars;	/\&* pointer to string */
+	int nchars;	/\&* number of bytes */
+	int delta;	/\&* pixel delta between strings */
+	XFontSet font_set; 	/\&* fonts, None means don't change */
 } XmbTextItem;
 .De
 The
@@ -241,10 +241,10 @@ structure contains:
 .TA .5i 2.5i
 .ta .5i 2.5i
 typedef struct {
-	wchar_t *chars;	/\(** pointer to wide char string */
-	int nchars;	/\(** number of wide characters */
-	int delta;	/\(** pixel delta between strings */
-	XFontSet font_set;	/\(** fonts, None means don't change */
+	wchar_t *chars;	/\&* pointer to wide char string */
+	int nchars;	/\&* number of wide characters */
+	int delta;	/\&* pixel delta between strings */
+	XFontSet font_set;	/\&* fonts, None means don't change */
 } XwcTextItem;
 .De
 .SH "SEE ALSO"
diff --git a/man/XmbTextListToTextProperty.man b/man/XmbTextListToTextProperty.man
index cad0cf3..23501e9 100644
--- a/man/XmbTextListToTextProperty.man
+++ b/man/XmbTextListToTextProperty.man
@@ -408,10 +408,10 @@ structure contains:
 .TA .5i 2.5i
 .ta .5i 2.5i
 typedef struct {
-	unsigned char *value;	/\(** property data */
-	Atom encoding;	/\(** type of property */
-	int format;	/\(** 8, 16, or 32 */
-	unsigned long nitems;	/\(** number of items in value */
+	unsigned char *value;	/\&* property data */
+	Atom encoding;	/\&* type of property */
+	int format;	/\&* 8, 16, or 32 */
+	unsigned long nitems;	/\&* number of items in value */
 } XTextProperty;
 .De
 .LP
@@ -447,11 +447,11 @@ T}
 .TA .5i 2.5i
 .ta .5i 2.5i
 typedef enum {
-	XStringStyle,		/\(** STRING */
-	XCompoundTextStyle,	/\(** COMPOUND_TEXT */
-	XTextStyle,		/\(** text in owner's encoding (current locale) */
-	XStdICCTextStyle,	/\(** STRING, else COMPOUND_TEXT */
-	XUTF8StringStyle	/\(** UTF8_STRING */
+	XStringStyle,		/\&* STRING */
+	XCompoundTextStyle,	/\&* COMPOUND_TEXT */
+	XTextStyle,		/\&* text in owner's encoding (current locale) */
+	XStdICCTextStyle,	/\&* STRING, else COMPOUND_TEXT */
+	XUTF8StringStyle	/\&* UTF8_STRING */
 } XICCEncodingStyle;
 .De
 .SH "SEE ALSO"
diff --git a/man/XrmInitialize.man b/man/XrmInitialize.man
index c282e61..c481bef 100644
--- a/man/XrmInitialize.man
+++ b/man/XrmInitialize.man
@@ -233,14 +233,14 @@ typedef struct {
 .TA .5i 2.5i
 .ta .5i 2.5i
 typedef enum {
-	XrmoptionNoArg,	/\(** Value is specified in XrmOptionDescRec.value */
-	XrmoptionIsArg,	/\(** Value is the option string itself */
-	XrmoptionStickyArg,	/\(** Value is characters immediately following option */
-	XrmoptionSepArg,	/\(** Value is next argument in argv */
-	XrmoptionResArg,	/\(** Resource and value in next argument in argv */
-	XrmoptionSkipArg,	/\(** Ignore this option and the next argument in argv */
-	XrmoptionSkipLine,	/\(** Ignore this option and the rest of argv */
-	XrmoptionSkipNArgs	/\(** Ignore this option and the next
+	XrmoptionNoArg,	/\&* Value is specified in XrmOptionDescRec.value */
+	XrmoptionIsArg,	/\&* Value is the option string itself */
+	XrmoptionStickyArg,	/\&* Value is characters immediately following option */
+	XrmoptionSepArg,	/\&* Value is next argument in argv */
+	XrmoptionResArg,	/\&* Resource and value in next argument in argv */
+	XrmoptionSkipArg,	/\&* Ignore this option and the next argument in argv */
+	XrmoptionSkipLine,	/\&* Ignore this option and the rest of argv */
+	XrmoptionSkipNArgs	/\&* Ignore this option and the next
 		\ \ \ XrmOptionDescRec.value arguments in argv */
 } XrmOptionKind;
 .De
@@ -249,10 +249,10 @@ typedef enum {
 .TA .5i 2.5i
 .ta .5i 2.5i
 typedef struct {
-	char *option;	/\(** Option specification string in argv		    */
-	char *specifier;	/\(** Binding and resource name (sans application name)    */
-	XrmOptionKind argKind;	/\(** Which style of option it is	    */
-	XPointer value;	/\(** Value to provide if XrmoptionNoArg or 
+	char *option;	/\&* Option specification string in argv		    */
+	char *specifier;	/\&* Binding and resource name (sans application name)    */
+	XrmOptionKind argKind;	/\&* Which style of option it is	    */
+	XPointer value;	/\&* Value to provide if XrmoptionNoArg or 
 		\ \ \ XrmoptionSkipNArgs   */
 } XrmOptionDescRec, *XrmOptionDescList;
 .De
-- 
1.4.1

From e3eb83ec6a9bffa63cdffd94f077c12f85ad7240 Mon Sep 17 00:00:00 2001
From: Alan Coopersmith <alan.coopersmith@sun.com>
Date: Mon, 11 Feb 2008 20:11:43 -0800
Subject: [PATCH] Spell out number in XkbGetKeyVirtualModMap man page to avoid cpp errors from #
---
 man/xkb/XkbGetKeyVirtualModMap.man |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/man/xkb/XkbGetKeyVirtualModMap.man b/man/xkb/XkbGetKeyVirtualModMap.man
index dcb7533..6b35a77 100644
--- a/man/xkb/XkbGetKeyVirtualModMap.man
+++ b/man/xkb/XkbGetKeyVirtualModMap.man
@@ -57,7 +57,7 @@ connection to server
 keycode of first key to fetch
 .TP
 .I \- num
-# keys for which virtual mod maps are desired
+number of keys for which virtual mod maps are desired
 .TP
 .I \- xkb
 Xkb description where results will be placed
-- 
1.4.1

